node/deny.toml

```
@@ -44,7 +44,7 @@ allow = [
 
 [bans]
 # Lint level for when multiple versions of the same crate are detected
-multiple-versions = "deny"
+multiple-versions = "warn"
```

```suggestion
multiple-versions = "deny"
```
So, just add it to the skip list below.

---

Fixed.